languages/English/English_global.php

```diff
@@ -2,10 +2,291 @@
 	/**
 	 * e107 website system
 	 *
-	 * Copyright (C) 2008-2016 e107 Inc (e107.org)
+	 * Copyright (C) 2008-2021 e107 Inc (e107.org)
 	 * Released under the terms and conditions of the
 	 * GNU General Public License (http://www.gnu.org/licenses/gpl.txt)
 	 *
 	 */
 
-define("LAN_PLUGIN_VSTORE_NAME", "Shop");
\ No newline at end of file
+define("LAN_PLUGIN_VSTORE_NAME", "Shop");
+define("LAN_PLUGIN_VSTORE_PRICE", "Price");
+
+define("LAN_VSTORE_DSTAT_001", "Open orders");
+define("LAN_VSTORE_DSTAT_002", "No Transactions Found");
+define("LAN_VSTORE_DSTAT_003", "Monthly Revenue: ");
+define("LAN_VSTORE_DSTAT_004", "Orders this month");
+define("LAN_VSTORE_DSTAT_005", "and");
+
+//define("LAN_VSTORE_PREF_001", "How to Order");// gen 30
+//define("LAN_VSTORE_008", "Re-Calculate");//gen 31
+//define("LAN_VSTORE_STAT_005", "Payed & Open orders");//gen 32
+//define("LAN_VSTORE_PROD_013", "Files"); //gen 33
+
+//define("LAN_VSTORE_STAT_DATA", "No orders available for the last week."); /gen34
+
+
+
+define("LAN_VSTORE_GEN_001", "Status");
```

Comment:
Please use the format `LAN_VSTORE_XXX` for frontend, and `LAN_VSTORE_ADMIN_XXX` for admin area. 
STAT, GEN etc, are not needed and no doubt some terms will be used in multiple areas.